data_browser/src/ExperimentPage.js

```diff
@@ -119,62 +125,305 @@ function renderExperimentHeader(args) {
+      </tbody>
+    </table>;
+    return {name: "consolidate", description};
+  }
+
+  // Train
+  if (step.fn_name === "marin.training.training.run_levanter_train_lm") {
+    return renderTrainStepDescription({step, steps});
+  }
+
+  // Evaluate
+  if (step.fn_name === "marin.evaluation.run.evaluate") {
+    return renderEvaluateStepDescription({step, steps, auxiliaryData});
+  }
+
+  return {name: step.fn_name, description: step.description};
+}
+
+function renderPath({path, steps}) {
+  // If path contains a pattern (e.g., gs://marin-us-central2/.../val*.jsonl.gz), strip those out
+  // This is what we link to since the data browser can't handle patterns
+  let linkedPath = path;
+  while (linkedPath.includes("*") || linkedPath.includes("{")) {
+    // Go up to the parent
+    const basename = linkedPath.split("/").pop();
+    linkedPath = linkedPath.substring(0, linkedPath.length - basename.length - 1);
+  }
+
+  // What to show
+  const replacedPath = replacePath({path, steps});
+
+  return <a href={viewSingleUrl(linkedPath)} target="_blank">{replacedPath}</a>;
+}
+    
+function replacePath({path, steps}) {
+  // If the path is under an output path of some step, then link to the name of that step
+  for (const step of steps) {
+    if (path.startsWith(step.output_path)) {
+      return path.replace(step.output_path, `[${step.name}]`);
+    }
+  }
+  return path;
+}
+
+function renderTokenizeStepDescription({step, steps}) {
+  const tokenizer = step.config.tokenizer;
+  const hfUrl = `https://huggingface.co/${tokenizer}/raw/main/tokenizer.json`;
+  const paths = step.config.train_paths.concat(step.config.validation_paths);
+  const links = paths.map((path, index) => {
+    return <div key={index}>{renderPath({path, steps})}</div>;
   });
-  const configTable = configRows.length === 0 ? null :
-    <table className="experiment-step-config"><tbody>{configRows}</tbody></table>;
-
-  // Map dependencies (represented by output paths) to the corresponding step indices
-  const outputPaths = experiment.steps.map(step => step.output_path);
-  const dependencies = step.dependencies.map(dep => `[${outputPaths.indexOf(dep)}]`).join(", ");
-
-  return (<Paper key={index} className="experiment-step">
-    <div>
-      Step [{index}]
-      <span className="description">{step.description}</span>
-    </div>
-    <div className="experiment-step-line"><a href={viewSingleUrl(step.output_path)}>{step.output_path}</a> := {step.fn_name}({dependencies})</div>
-    {configTable}
-    {renderExperimentStatus(events)}
-    <ButtonGroup>
-      {links.map((link, i) => <span key={i}>{link}</span>)}
-    </ButtonGroup>
-  </Paper>);
+  const description = <table>
+    <tbody>
+      <tr><td>Tokenizer:</td><td><a href={hfUrl} target="_blank">{tokenizer}</a></td></tr>
+      <tr><td>Raw text:</td><td>{links}</td></tr>
+    </tbody>
+  </table>;
+  return {name: "tokenize", description};
+}
+
+function renderTrainStepDescription({step, steps}) {
```

Comment:
@dlwh maybe take a look at this to see if this is showing reasonable stuff for a training run